packages/console/src/components/PlanGate.tsx

```diff
@@ -20,10 +21,16 @@ const PricingTable = ({
   const { isIframe } = useIframe()
   
   return (
-    <div className="flex flex-col justify-center items-center min-h-screen">
+    <div className="flex flex-col justify-center items-center min-h-screen relative">
       <div className="my-6">
         <Logo />
       </div>
+      <Link
+        href="https://storacha.console.network/logout"
```

Comment:
this URL should be different per-environment - in fact, does it even need to be an absolute URL? why not use a relative URL here?